backend/src/v4/models/invitations.js

```diff
@@ -34,7 +34,7 @@ const MODELS_PERMISSION = ["collaborator", "commenter", "viewer"];
 
 const getCollection = async () => {
 	const coll = await db.getCollection("admin", "invitations");
-	coll.createIndex({ "teamSpaces.teamspace": 1 }, { "background": true, "w":1 });
+	coll.createIndex({ "teamSpaces.teamspace": 1 }, { "background": true, writeConcern: { w: "1" } });
```

Comment:
@sanmont3drepo  I did a git blame and this is your code... 😆 

Do you remember why we are trying to create an index everytime we do getCollection here? and why did you have to specify write concern?

Reply:
I Dont remember. If I have to guess it was probably because a warning or an error that I got if that parameter wasnt there.